docs/user-guides/user-guide-gui.md

```diff
@@ -70,6 +70,16 @@ Paste the pull command into your terminal to pull the image.
 
 > :bulb: For information about using image management tools to pull images, see [Push and Pull Image Content](user-guide-datapath.md).
 
+<a name="bookmarks"></a>
+## Adding Bookmarks
+In order to save certain images important to the user the bookmark feature can be used. When the bookmark icon is pressed the image is marked as such and it will appear in search queries when the Bookmarked option is enabled.
```

Comment:
Edit: "To mark an image so that it can be easily found later, click the bookmark icon on the image page. Bookmarked images will appear in search queries when the Bookmarked option is enabled."